### ISSUE TITLE
Include cmd/client-stats in the release process

### ISSUE THREAD

**Issue by kasey** [prysmaticlabs/prysm#8908]
# 🚀 Feature Request

### Description

We are not currently producing a binary for the client-stats cli (https://github.com/prysmaticlabs/prysm/tree/develop/cmd/client-stats).

### Describe the solution you'd like

Add a build target for this binary and include it in the release process.

cc @prestonvanloon

**Comment by yorickdowne** [comment]
Yes please. It'd be awesome to have this available in docker hub.

**Comment by prestonvanloon** [comment]
The only remaining item for this is to add docker targets.
